guava/src/com/google/common/base/CaseFormat.java

```diff
@@ -133,6 +133,9 @@ public final String to(CaseFormat format, String str) {
   @SuppressWarnings({
     "upperbound:assignment.type.incompatible", "upperbound:compound.assignment.type.incompatible", // refinement on multiple assignments in expression
     "upperbound:argument.type.incompatible", // length of sequence with ArrayLenRange in offset
+    "dereference.of.nullable" // Variable 'out' will never be dereferenced when it is null. If block
```

Comment:
How about moving the initialization of `out` from line 145 to line 139 where it is declared?
Then I think the suppression is not necessary.